GraphQL-TSC.md

```diff
@@ -200,16 +200,34 @@ Note: A member may be recused (i.e. for a member election) in which case they do
 
 ### Voting process
 
-Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. Once the threshold of a quorum has been met and a vote is valid, one of these two critera must be satisfied to conclude a vote:
+The GraphQL TSC is a distributed group and our voting process must account for a range of time zones and schedules. TSC votes are held asynchronously and follow three distinct phases:
+
+1. Call for vote
```

Comment:
Capturing a note from @benjie - need to capture what constitute a vote, and what appears similar but is different (for example, an opinion poll or asking for feedback)